numpy_challenge.py

```diff
@@ -0,0 +1,52 @@
+import numpy as np
+
+
+def matrix_multiplication(matrix1, matrix2):
+    return np.matmul(matrix1, matrix2)
+
+
+def multiplication_check(matrices):
+    starting_matrix = matrices[0]
+    if np.isscalar(starting_matrix[0]):
+        prev_matrix_columns = 1
+    else:
+        prev_matrix_columns = len(starting_matrix[0])
+    for i in range(1, len(matrices)):
+        current_matrix = matrices[i]
+        curr_matrix_rows = len(current_matrix)
+        if prev_matrix_columns != curr_matrix_rows:
+            return False
+        if np.isscalar(starting_matrix[0]):
+            prev_matrix_columns = 1
+        else:
+            prev_matrix_columns = len(starting_matrix[0])
```

Comment:
Ошибочка) Кол-во колонок и предыдущей матрицы нужно задавать по текущей, а не по стартовой
```suggestion
            prev_matrix_columns = len(current_matrix[0])
```